core/attestation.go

```diff
@@ -16,6 +17,21 @@ type G1Point struct {
 	*bn254.G1Affine
 }
 
+func newFpElement(x *big.Int) fp.Element {
+	var p fp.Element
+	p.SetBigInt(x)
+	return p
+}
+
+func NewG1Point(x, y *big.Int) *G1Point {
```

Comment:
Not used here, is it for the other PR? If so maybe move it over there.

Reply:
Used here: https://github.com/mooselumph/eigenda/blob/accfb672699d22282dc1a1ee8d32e4106a519bd3/core/eth/tx.go#L116